main.py

```diff
@@ -62,8 +63,7 @@ def main(WorldClass):
             clock.tick(FRAME_RATE)
             display.flip()
             pygame.display.set_caption(TITLE + ' ' + world.get_info())
-        world.end(state_pattern)
-
+        world.end(constants.state_pattern)
```

Comment:
Ik denk dus eigenlijk dat we dit met een flag moeten doen, maar ik kan dat ook later doen.